assets/sass/_variables.scss

```diff
@@ -61,4 +63,5 @@ html[data-theme="dark"] {
   --highlight-color-tip: #1A4A5C;
   --highlight-color-inofficial: #2A2D30;
   --highlight-color-important: #4A2A1A;
+  --border: .1rem solid #3d444d;
```

Comment:
What do you think of removing the border? The border with the color is quite noticeable. If the contrast is not good enough, maybe we can use a slightly darker bg-neutral and a slightly lighter bg-default?

If we decide to keep the border, it should also be applied to the operator and country overview pages.